File: Code Boiler/golang/Analytics.go
package main

import "fmt"

type Analytics struct {
	paymentTransactions map[string]float64
}

func (a *Analytics) AddPaymentTransaction(item string, amount float64) {
	a.paymentTransactions[item] = amount
}

func (a *Analytics) GetPaymentTransactions() map[string]float64 {
	return a.paymentTransactions
}

func main() {
	analytics := &Analytics{paymentTransactions: make(map[string]float64)}
	analytics.AddPaymentTransaction("item1", 10.50)
	analytics.AddPaymentTransaction("item2", 15.25)
	fmt.Println(analytics.GetPaymentTransactions())
}

File: signup/Java/Analytics.java
package signup.Java;


import java.util.HashMap;
import java.util.Map;

public class Analytics{
    private Map<String, Double> paymentTransactions;

    public Analytics() {
        paymentTransactions = new HashMap<>();
    }

    public void addPaymentTransaction(String item, double amount) {
        paymentTransactions.put(item, amount);
    }

    public Map<String, Double> getPaymentTransactions() {
        return paymentTransactions;
    }
}
